src/Features/Core/Portable/MetadataAsSource/AbstractMetadataAsSourceService.AbstractWrappedSymbol.cs

```diff
@@ -120,7 +120,7 @@ public bool Equals(ISymbol other)
 
             public bool Equals(ISymbol other, SymbolEqualityComparer equalityComparer)
             {
-                return equalityComparer.Equals(this, other);
+                return Equals(other);
```

Comment:
>Equals(other); [](start = 23, length = 14)

Is this reference equality now?

Reply:
> Is this reference equality now? 

I believe this calls the method above, which delegates to object equality. I believe SymbolEqualityComparer used to do the same thing.

---
In reply to: [339191218](https://github.com/dotnet/roslyn/pull/39498#discussion_r339191218) [](ancestors = 339191218)